src/main/java/com/truelayer/java/entities/ProviderAvailability.java

```diff
@@ -0,0 +1,17 @@
+package com.truelayer.java.entities;
+
+import java.time.ZonedDateTime;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+@Builder
+@Getter
+@EqualsAndHashCode
+@ToString
+public class ProviderAvailability {
+    String recommendedStatus;
```

Comment:
Not sure if we want to use an enum here. We should check what did elsewhere. 
Pro: more close to developers
Cons: Potentially diverging from the API

Reply:
Yeah the reason why I opted for a string was to avoid issues in case those values will change in the future, but I can see how an enum could be more useful for developers integrating the SDK. It looks like we used enums in other similar situations like transaction type and status, so I'll switch to an enum